app/cp_nigeria/helpers.py

```diff
@@ -1389,7 +1395,7 @@ def collect_financial_params(self):
             "estimated_tariff",
         )
         qs_ed = EconomicData.objects.filter(project=self.project).values("discount", "tax")
-        financial_params = qs_ed.first() | qs_eq.first()
+        financial_params = {**qs_ed.first(), **qs_eq.first()}
```

Comment:
what is the difference here between the old line and the new one? don't they both merge the dictionaries?

Reply:
This is wrongly committed, the difference is that my version is compatible with python 3.8 ^^, I used to always stash it and I commited it by mistake, my bad